quadrat/block-template-parts/page.html

```diff
@@ -1,5 +1,19 @@
 <!-- wp:template-part {"slug":"header"} /-->
 
+<!-- wp:group {"style":{"spacing":{"padding":{"top":"30px","right":"20px","bottom":"60px","left":"20px"}}},"layout":{"inherit":true}} -->
+<div class="wp-block-group" style="padding-top:30px;padding-right:20px;padding-bottom:60px;padding-left:20px">
+	<!-- wp:post-title {"textAlign":"center","level":1,"align":"wide"} /-->
```

Comment:
@beafialho @kjellr what should the treatment for page titles be? For now I copied what we have for posts — center the text + wide alignment:

<img width="1904" alt="Screen Shot 2021-06-28 at 2 11 19 PM" src="https://user-images.githubusercontent.com/5375500/123683906-a3d2cc00-d801-11eb-8301-f01b27b03257.png">

Reply:
Keeping it the same as posts makes sense.